src/prefect/backend/kv_store.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any, List
+
+import prefect
+from prefect.utilities.graphql import with_args
+from prefect.client import Client
+from prefect.utilities.exceptions import ClientError
+
+
+def set_key_value(key: str, value: Any) -> str:
+    """
+    Set key value pair, overwriting values for existing key
+
+    Args:
+        - key (str): the name of the key
+        - value (Any): A json compatible value
+
+    Returns:
+        - id (str): the id of the key value pair
+
+    Raises:
+        - ClientError: if using Prefect Server instead of Cloud
+    """
+    if prefect.config.backend != "cloud":
```

Comment:
again a bit of boilerplate here for all kv functions, would a decorator be overkill?

Reply:
Let's save the decorator for later when there are more backend functions that are Cloud only